RobotPose robot_pose
string robot_pose_name
---
bool success
